src/hooks/useSignForWalletDerivation.tsx

```diff
@@ -13,12 +15,18 @@ export default function useSignForWalletDerivation() {
   const chainId = Number(ethereumChainId);
 
   const signTypedData = getSignTypedData(selectedDydxChainId);
-  const { signTypedDataAsync } = useSignTypedData({
-    ...signTypedData,
-    domain: {
-      ...signTypedData.domain,
-      chainId,
-    },
-  });
-  return signTypedDataAsync;
+  const { signTypedDataAsync } = useSignTypedData();
```

Comment:
wagmi 2 hooks don't take in params for the most part anymore. Params are passed into the function directly